pkg/infra/pulumi_aws/iac/sanitization/aws/elb.ts

```diff
@@ -56,3 +56,12 @@ export const targetGroup = {
         }
     },
 }
+
+console.log(
+    (
```

Comment:
Just out of curiosity, what does the extra parenthesis do? (It's not a problem, I'm just wondering if there's something neat going on I don't know :) )

Reply:
🤦 I accidentally committed the line of code I was using to reproduce the issue. That was just  a workaround to be able to invoke the fix function, which TypeScript complains about on its own because it _might be_ undefined.

Reply:
Can use `!` for cases where you know better than the compiler to assert that it isn't undefined.